packages/compass-collection/src/stores/index.ts

```diff
@@ -71,11 +71,10 @@ export const INITIAL_STATE = {
 
 /**
  * Handle the reset.
- *
- * @returns {Object} The new state.
  */
-const doReset = () => ({
+const doReset = ({ appRegistry }) => ({
   ...INITIAL_STATE,
+  appRegistry,
```

Comment:
Where do we send this app registry? Is it being set to undefined?

Reply:
When the compass bootstraps, we setup all plugins. In `mongodb-compass` plugin we call `global.hadronApp.appRegistry.onActivated();` which calls `onActivated` for every store of a plugin. And that's the reason we are not setting it null here.